app/models/consultation.rb

```diff
@@ -170,4 +171,10 @@ def validate_closes_after_opens
       errors.add :closing_at, "must be after the opening on date"
     end
   end
+
+  def validate_consultation_principles
+    unless read_consultation_principles
+      errors.add :read_consultation_principles, ""
```

Comment:
Then the error message is

> Unable to force_publish this edition because it is invalid (Read consultation principles Must have read the consultation principles). Please edit it and try again.

Reply:
Ugh 😒 I guess we could have "must be ticked" in the string which might read slightly nicer.